lib/montage/query.rb

```diff
@@ -77,37 +101,60 @@ def order(clause = {})
 
     # Adds a where clause to the query filter hash
     #
-    # Accepts either a Hash or a String
-    #     where(foo: 1)
-    #     where("foo > 1")
-    #
-    # Merges a hash:
-    #   { foo: 1 }
-    #
-    # Returns a copy of self
+    # * *Args* :
+    #   - +clause+ -> A hash or string containing desired options
+    # * *Example* :
+    #   - where(foo: 1)
+    #   - where("foo > 1")
+    # * *Returns* :
+    #   - A copy of self
     #
     def where(clause)
-      clone.tap { |r| r.query[:filter].merge!(QueryParser.new(clause).parse) }
+      clone.tap { |r| r.query["$query"]["$filter"].merge!(QueryParser.new(clause).parse) }
```

Comment:
I don't think we should start implementing these changes in this PR. Please revert it.